solr/solrj/src/java/org/apache/solr/client/solrj/impl/InputStreamResponseParser.java

```diff
@@ -44,4 +51,12 @@ public NamedList<Object> processResponse(Reader reader) {
   public NamedList<Object> processResponse(InputStream body, String encoding) {
     throw new UnsupportedOperationException();
   }
+
+  public static NamedList<Object> createInputStreamNamedList(
+      int httpStatus, InputStream inputStream) {
+    final var nl = new NamedList<Object>();
```

Comment:
Please always use a SimpleOrderedMap when the keys don't repeat

Reply:
Done!